pylint/config.py

```diff
@@ -447,7 +447,8 @@ def _convert_definition(option, definition):
 
         args.append('--{0}'.format(option))
 
-        copy_keys = ('action', 'default', 'dest', 'help', 'metavar', 'level')
+        copy_keys = (
+            'action', 'default', 'dest', 'help', 'metavar', 'level', 'version')
```

Comment:
`version` is a special kwarg for add_argument when using the `'version'` action.